### PR TITLE
Integrate Chronicle logging for errors

### PR DESCRIPTION
**Pull request by aliciaxw**
Logs server-side errors to Amazon S3 if on production environment. Requires two new environment variables for Chronicle (see updated README). #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ CREATE DATABASE pollo;
 
 ### Required variables:
 ````bash
+export CHRONICLE_ACCESS_KEY=FILL_IN
```
Comment (young-k) [inline README.md]: can we also add an `env.template` file with the environment variables, refer to: https://github.com/cuappdev/uplift-backend

File: src/utils/LogUtils.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Error.captureStackTrace(obj, getStackTrace);
+        return obj.stack;
+    });
+}
+
+/**
+ * Log error to Chronicle if production
+ * @param {string} msg The error message
+ */
+function logErr(msg: string) {
```
Comment (young-k) [inline src/utils/LogUtils.js]: can we change this to `logError`

**Comment by aliciaxw** [comment]
I was testing with an arbitrary error to make sure the logging to S3 worked correctly (it was a bit difficult to reproduce existing errors so I just temporary hit some endpoint with logErr), so that's why you see `testing testing 123`.

And yes, I did do a global search and a replace. Should I not have? 😮

**Comment by aliciaxw** [comment]
@young-k updated logErr -> logError and added an env.template!

**Pull request merged**